ci/test_utils.sh

```diff
@@ -20,5 +20,8 @@ print_system_stats() {
 
 print_ucx_config() {
   rapids-logger "UCX Version and Build Configuration"
+
+  set +e
   ucx_info -v
+  set -e
```

Comment:
Are these change relevant? Moreover, are they still necessary now that the wheel test scripts are no longer calling `print_ucx_config`?

Reply:
You're right, we don't need it anymore. Removed in https://github.com/rapidsai/ucxx/pull/167/commits/035c45352f9dd23c717bc5c66811be8652b2a591